.github/CODEOWNERS

```
@@ -37,7 +37,7 @@
 
 # Translations (i18n)
 airflow-core/src/airflow/ui/src/i18n/locales/de/ @jscheffl # not codeowner but engaged: @TJaniF @m1racoli
-airflow-core/src/airflow/ui/src/i18n/locales/zh-TW/ @Lee-W
+airflow-core/src/airflow/ui/src/i18n/locales/zh-TW/ @Lee-W @jason810496 # not codeowner but engaged: @RoyLee1224 @guan404ming
```

As I just merged #51730:
```suggestion
airflow-core/src/airflow/ui/src/i18n/locales/zh-TW/ @Lee-W @jason810496 # + @RoyLee1224 @guan404ming
```